### PR TITLE
Lbibass aek clueboard

### PR DESCRIPTION
**Pull request by lbibass**
## Description

revised my keymap, and fixed a glitch in the AEK matrix that swapped the keys in the p and [{ columns. 

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clueboard/60/keymaps/lbibass_aek/keymap.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#include "60.h"
```
Comment (fauxpark) [inline keyboards/clueboard/60/keymaps/lbibass_aek/keymap.c]:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/clueboard/60/60.h [hunk 2]
```diff
@@ -45,17 +45,17 @@
  * 3 mods on either side of the spacebar.
  */
 #define LAYOUT_aek(\
-    k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c,   k0e, \
-     k10,   k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c,   k1e, \
-     k20,    k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b,    k2e, \
-       k30,   k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b,      k3c, \
+    k00, k01, k02, k03, k04, k05, k06, k07, k08, k09,  k0b, k0a, k0c,   k0e, \
+     k10,   k11, k12, k13, k14, k15, k16, k17, k18, k19, k1b,k1a,  k1c,   k1e, \
```
Comment (drashna) [inline keyboards/clueboard/60/60.h]: @skullydazed any issues with this change?

Comment (skullydazed) [reply keyboards/clueboard/60/60.h]: Yes, I don't understand why this change is here. I'm looking at the pcb design and the original file should be correct?

Comment (drashna) [reply keyboards/clueboard/60/60.h]: well, that's why I tagged you. 

>revised my keymap, and fixed a glitch in the AEK matrix that swapped the keys in the p and [{ columns.

As reported by @lbibass.  As I don't have the board, I can't even begin to confirm it.

Comment (lbibass) [reply keyboards/clueboard/60/60.h]: yeah, I wasn't sure what was going on, but when I flashed it, those two rows were flipped. I am not sure what caused it, but doing what I did fixed it. Do you mind testing it on your end? I can't think of anything on my end that would have caused that, but I could be wrong.

Comment (tzarc) [reply keyboards/clueboard/60/60.h:49]: @skullydazed did you ever get a chance to confirm this? Would be great to get this PR modified or merged, considering the lifetime.

**Comment by tzarc** [comment]
Still needs confirmation from @skullydazed -- if it's critical this gets merged it can be chased up. Stalebot will kick in, otherwise.

File: keyboards/clueboard/60/keymaps/lbibass_aek/rules.mk [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+## (Note that for BOOTMAGIC on Teensy LC you have to use a custom .ld script.)
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+#SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
+CUSTOM_MATRIX = no # Custom matrix file
+AUDIO_ENABLE = yes
+# SERIAL_LINK_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/clueboard/60/keymaps/lbibass_aek/rules.mk:12]:
```suggestion
BOOTMAGIC_ENABLE = yes
COMMAND_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
Thank you for your contribution!

This pull request has been automatically closed because it has not had activity in the last 30 days.
Please feel free to give a status update now, ping for review, or re-open when it's ready.

**Pull request closed without merging**